statsdaemon.go

```diff
@@ -269,12 +270,17 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 	for g, c := range gauges {
 		lastValue, ok := trackedGauges[g]
 
-		if ok && c == lastValue {
+		if ok && c == math.MaxUint64 && *deleteGauges {
 			continue
+		} else if ok && lastValue > 0 && c == math.MaxUint64 && *deleteGauges == false {
```

Comment:
also i'm thinking that `s/trackedGauges/lastGaugeValue` might be a better variable name.